CHANGELOG_PENDING.md

```diff
@@ -8,158 +8,9 @@ Friendly reminder: We have a [bug bounty program](https://hackerone.com/tendermi
 
 ### BREAKING CHANGES
 
-- CLI/RPC/Config
```

Comment:
Do we want to keep these sub-categories

Reply:
yes! They were in v0.34 as well in the `CHANGELOG_PENDING.md`. Re-added